spring-boot-samples/spring-boot-sample-kafka/src/main/java/sample/kafka/Consumer.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2012-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sample.kafka;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+@Service
+public class Consumer {
+	private static Gson gson = new GsonBuilder().create();
+
+	@KafkaListener(topics = "myTopic")
+	public void onMessage(String message) {
```

Comment:
Rather that parsing  the json yourself, it would be nice  to configure Kafka to do that for us automatically.